src/crimson/os/seastore/seastore_types.h

```diff
@@ -347,6 +347,25 @@ enum class extent_types_t : uint8_t {
   NONE = 0xFF
 };
 
+// FIXME: reassign extent_types_t values instead
+inline uint8_t extent_type_to_index(extent_types_t type) {
```

Comment:
Can I reassign the values of `extent_types_t` so that it is easier to index them in array? @athanatos

Reply:
see https://github.com/ceph/ceph/pull/42539/commits/d0fa5e2a9d425534640859edd23cba842a768f49